src/components/CommentCard.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export default function CommentCard() {
```

Comment:
Since this is a card, it should be receiving the comment object as an input.That would avoid the hard coding of input value and make the application more dynamic which makes integration very fast

Reply:
Solved.